### PR TITLE
Added list of products

### PR DESCRIPTION
**Pull request by micaelpimentel**
- List of products with recycler view
- Implemented recycler view adapter

### CODE REVIEW

**Comment by micaelpimentel** [comment]
WhatsApp Image 2021-09-29 at 6 23 10 PM

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -6,15 +6,28 @@
...
+            <androidx.recyclerview.widget.RecyclerView
+                android:id="@+id/recyclerView"
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_margin="15dp"
+                app:layoutManager="androidx.recyclerview.widget.LinearLayoutManager"
+                tools:itemCount="10"
+                tools:listitem="@layout/list_item" />
+        </androidx.constraintlayout.widget.ConstraintLayout>
+    </androidx.core.widget.NestedScrollView>
```
Comment (MathBrandino) [inline app/src/main/res/layout/activity_main.xml:30]: i don't understand the reason that you need to use the NestedScrollView in this case, can you explain me the reason ?

File: app/src/main/res/layout/list_item.xml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        tools:text="Cesta de Frutas" />
+
+    <TextView
+        android:id="@+id/productDescriptionTextView"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/productNameTextView"
+        tools:text="Várias frutas na cesta" />
```
Comment (MathBrandino) [inline app/src/main/res/layout/list_item.xml:25]: nice approach to use the tools namespace to see in xml the preview 👍

**Review by MathBrandino — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/activity_main.xml (hunk 1 above)
Comment (micaelpimentel) [reply app/src/main/res/layout/activity_main.xml:30]: I put a scroll here so the list can be scrolled when there are too many itens to be displayed. I thought it made sense

**Pull request merged**